AmericanMedicalAssociation/ama-style-guide-2
# EWL-9037: Adds homepage social module designs to style guide.

## Ticket(s)
**Do not** submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.

**Github Issue**
- N/A

**Jira Ticket**
- [EWL-9037: Home Page Social Module Styleguide](https://issues.ama-assn.org/browse/EWL-9037)

## Description
Adds mock ups for the new Homepage Social module to the styleguide.


## To Test
- Open styleguide locally (gulp serve)
- Under the 'Organisms' tab select the Homepage Social Module
- Confirm both the standard and 'three-up' version match the following Zeplins:

[Desktop](https://zpl.io/anzXkjk)
[Tablet](https://zpl.io/2Z3dDR4)
[Mobile](https://zpl.io/2yo563w)


## Visual Regressions
- N/A

## Relevant Screenshots/GIFs
![Screen Shot 2021-09-07 at 11 08 49 AM](https://user-images.githubusercontent.com/67962801/132377346-b68044eb-8f0b-45df-a922-8d7598212db5.png)
![Screen Shot 2021-09-07 at 11 09 10 AM](https://user-images.githubusercontent.com/67962801/132377349-b3327c0e-6bdd-41a3-bffc-f228aa92f105.png)
![Screen Shot 2021-09-07 at 11 09 16 AM](https://user-images.githubusercontent.com/67962801/132377356-c4108b18-37cd-43ca-bdde-64f6c5972fab.png)


## Remaining Tasks
- N/A


## Additional Notes
- N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
